scripts/us_census/fred/homeownership/README.md

```diff
@@ -0,0 +1,4 @@
+# Scripts for importing datasets from FRED data on Kaggle
+
+This directory stores all scripts used to import datasets into Data Commons from FRED (Federal Reserve Economic Data, from the St. Louis Fed) via the
+US Census Bureau's account on Kaggle.
```

Comment:
Can you add some links. Also here is an example README (feel free to make suggestions to this template): https://github.com/datacommonsorg/data/tree/master/scripts/example_provenance/example_dataset

Reply:
Hi @tjann, thanks for the example README! I followed it pretty closely and anticipate linking to it in future documentation.

In the process of adding documentation, I realized that the CSV contained two columns that were not used in the final knowledge graph, so I went ahead and removed them. I know that changes this PR's scope a bit--if you'd like me to move those changes to another PR, let me know!

Reply:
@KilimAnnejaro thanks! Feel free to improve upon it too!

Yeah, could you revert that change? It's totally ok for the CSV to have more columns than used. There's no need to remove them, and is a selling point here (less pre-processing).

Reply:
@tjann reverted!